### PR TITLE
feat: distinguish API URL from web app URL in the CLI

### PR DESCRIPTION
**Pull request by guilgaly**

### CODE REVIEW

File: jvm/build.sbt
Comment (guilgaly) [inline jvm/build.sbt:22]: Waiting for release before merging

File: js/cli/src/commands/options.ts
Comment (guilgaly) [inline js/cli/src/commands/options.ts:218]: Not backwards-compatible (changes the CLI option), but this setting was never documented for end users (`.hideHelp()`).

**Review by notdryft — APPROVED** [review]

**Pull request merged**
